package.json

```diff
@@ -246,6 +248,7 @@
     "secp256k1@npm:^4.0.1": "4.0.4",
     "secp256k1@npm:4.0.2": "4.0.4",
     "secp256k1@npm:4.0.3": "4.0.4",
+    "secp256k1@npm:5.0.0": "5.0.1",
```

Comment:
?

Reply:
Good question! Looking now

Reply:
This seems likely a rebase issue.

Reply:
![Screenshot 2025-01-31 at 9 50 22 AM (2)](https://github.com/user-attachments/assets/918c5e22-9580-4977-bdb3-6da9a77d5673)

So this was the reason for the bump here... I'm still not clear why this is only showing up on this PR tho

Reply:
Unclear to me why this version bump is necessary for this PR

<img width="794" alt="Screenshot 2025-01-31 at 11 53 01 AM" src="https://github.com/user-attachments/assets/4fa24767-70b6-472c-a561-f16548cf0b8d" />

Reply:
Not sure why the `yarn.lock` file got so bloated, but seems like the audit failure that was happening because we were bumping `gridplus-sdk` a couple minor versions is gone now since we don't actually need to do that bump in this PR: https://github.com/MetaMask/metamask-extension/pull/27782/commits/a1391244b4588908154be956fe8281830e71a895